### PR TITLE
Support to FormData (web)

### PR DESCRIPTION
**Pull request by thiagosantos**
Adding support to FormData and set it as default when no content-type is defined but having data into options.data, forcing the browser to do 'MIME Sniffing'.

### CODE REVIEW

**Comment by muuvmuuv** [comment]
LGTM, what about Android and Swift implementation?

**Comment by thomasvidas** [comment]
I _believe_ that Android and iOS already support this [here (Android)](https://github.com/capacitor-community/http/blob/ab60876c1784eb0074b3b1857c13fe5d57089617/android/src/main/java/com/getcapacitor/plugin/http/Http.java#L482) and [here (iOS)](https://github.com/capacitor-community/http/blob/ab60876c1784eb0074b3b1857c13fe5d57089617/ios/Plugin/Plugin.swift#L351) so its only missing on web if I'm reading the code correctly.

I'll add it to my list of things to review 👍

**Comment by thiagosantos** [comment]
@muuvmuuv sorry for the delayed response, I miss this.
@thomasvidas that's right, this feature is lacking only for the web.

best!

File: src/web.ts [hunk 1]
```diff
@@ -62,15 +62,29 @@ export class HttpPluginWeb extends WebPlugin implements HttpPlugin {
...
+      req['body'] = formData;
+    }else{
+      if(typeof options.data == 'object'){
+        let formData = new FormData();
+        for (let key of Object.keys(options.data)){
+          formData.append(key, options.data[key])
+        }
+        req['body'] = formData;
+      }
     }
```
Comment (thomasvidas) [inline src/web.ts:86]: Can you combine the `typeof options.data == 'object'` and `contentType.indexOf('multipart/form-data) === 0` into a single `if` block?

File: src/web.ts [hunk 2]
```diff
@@ -62,15 +62,29 @@ export class HttpPluginWeb extends WebPlugin implements HttpPlugin {
...
       }
       req['body'] = urlSearchParams.toString();
+    } else if(contentType.indexOf('multipart/form-data') === 0 ){
+      let formData = new FormData();
+      for (let key of Object.keys(options.data)){
+        formData.append(key, options.data[key])
+      }
+      req['body'] = formData;
+    }else{
+      if(typeof options.data == 'object'){
```
Comment (thomasvidas) [inline src/web.ts:79]: `===` rather than `==`

**Review by thomasvidas — CHANGES_REQUESTED** [review]
Just a few, minor changes @thiagosantos but looks good! I'll merge it in once you make the changes 😄

File: src/web.ts (hunk 1 above)
Comment (thiagosantos) [reply src/web.ts]: Thomas I just made a commit!

**Review by thiagosantos — COMMENTED** [review]
Merge two blocks into one and an exactly equal comparation.

**Pull request merged**